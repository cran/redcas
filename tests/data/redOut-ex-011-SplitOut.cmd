4: %% Test with echo off
4: off echo ;
5: 
6: 
7: 
8: 
9: 9: 9: 
10: b:=(u+v)^3 ;
11: bu:=df(b,u) ;
12: bv:=df(b,v) ;
13: mark_end_block(2) ;
14: c:=(u+v)^4 ;
15: cu:=df(c,u) ;
16: cv:=df(c,v) ;
17: mark_end_block(3) ;
18: 
