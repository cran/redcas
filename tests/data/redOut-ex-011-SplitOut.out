      2            2
a := x  + 2*x*y + y

ax := 2*(x + y)

ay := 2*(x + y)

=====> submit number 1 done <=====


      3      2          2    3
b := u  + 3*u *v + 3*u*v  + v


          2            2
bu := 3*(u  + 2*u*v + v )


          2            2
bv := 3*(u  + 2*u*v + v )


=====> submit number 2 done <=====


      4      3        2  2        3    4
c := u  + 4*u *v + 6*u *v  + 4*u*v  + v


          3      2          2    3
cu := 4*(u  + 3*u *v + 3*u*v  + v )


          3      2          2    3
cv := 4*(u  + 3*u *v + 3*u*v  + v )


=====> submit number 3 done <=====



